test/simulator/timestamp/src/include/session_simulator.h

```diff
@@ -33,9 +33,14 @@ class session_simulator {
     public:
     session_simulator();
```

Comment:
Do we need the default constructor/destructor? Since we don't do anything specific with them, we probably don't need those lines.

Reply:
I guess so, I'm not sure if we plan on using them.

Reply:
We need the constructor, we use for conn->open_session.